src/environments/environment.stg.ts

```diff
@@ -23,5 +23,6 @@ export const environment = {
   SENTRY_ENVIRONMENT: 'staging',
   idleTime: 900,
   idleTimeout: 300,
+  orgRequestEmail: 'john.henderson@energyweb.org',
```

Comment:
I think devops will make the email soon, so I'm betting that this will avoid needing to change it down the road
```suggestion
  orgRequestEmail: 'iamteam@energyweb.org@energyweb.org',
```

Reply:
if you prefer we can always move it to gh variables :)